doc/about.rst

```diff
@@ -96,86 +96,176 @@ following paper:
 Artwork
 -------
 
-High quality PNG and SVG logos are available in the `doc/logos/ <https://github.com/scikit-learn/scikit-learn/tree/master/doc/logos>`_ source directory.
+High quality PNG and SVG logos are available in the `doc/logos/
+<https://github.com/scikit-learn/scikit-learn/tree/master/doc/logos>`_
+source directory.
 
 .. image:: images/scikit-learn-logo-notext.png
    :align: center
 
 Funding
 -------
+Scikit-Learn is a community driven project, however institutional and private
+grants help to assure its sustainability.
 
-`INRIA <https://www.inria.fr>`_ actively supports this project. It has
-provided funding for Fabian Pedregosa (2010-2012), Jaques Grobler
-(2012-2013) and Olivier Grisel (2013-2017) to work on this project
-full-time. It also hosts coding sprints and other events.
+The project would like to thank the following funders. 
 
-.. image:: images/inria-logo.jpg
-   :width: 200pt
-   :align: center
-   :target: https://www.inria.fr
+...................................
 
-`Paris-Saclay Center for Data Science <https://www.datascience-paris-saclay.fr/>`_
-funded one year for a developer to work on the project full-time
-(2014-2015) and 50% of the time of Guillaume Lemaitre (2016-2017).
+The `Scikit-Learn Consortium at Inria Foundation
+<https://scikit-learn.fondation-inria.fr/en/home/>`_ `Members 
+<https://scikit-learn.fondation-inria.fr/en/home/#sponsors>`_ fund Olivier
+Grisel, Guillaume Lemaitre, Jérémie Du Boisberranger and Chiara Marmo.
 
-.. image:: images/cds-logo.png
-   :width: 200pt
-   :align: center
-   :target: https://www.datascience-paris-saclay.fr/
+**Platinum Members**
 
-`NYU Moore-Sloan Data Science Environment <https://cds.nyu.edu/mooresloan/>`_
-funded Andreas Mueller (2014-2016) to work on this project. The Moore-Sloan Data Science
-Environment also funds several students to work on the project part-time.
 
-.. image:: images/nyu_short_color.png
-   :width: 200pt
+.. figure:: images/microsoft.png
+   :width: 100pt
    :align: center
-   :target: https://cds.nyu.edu/mooresloan/
+   :target: https://www.microsoft.com/
 
+.. figure:: images/bcg.png
+   :width: 100pt
+   :align: center
+   :target: https://www.bcg.com/beyond-consulting/bcg-gamma/default.aspx
```

Comment:
I think we should find a way to pack the icons of the sponsors of the sckit-learn consortium on 3 lines (by sponsoring levels). I find that the current layout takes to much vertical real estate.